src/moin/scripts/migration/moin19/import19.py

```diff
@@ -235,10 +236,15 @@ def iter_revisions(self):
             fnames = os.listdir(revisionspath)
         except OSError:
             fnames = []
+        parent_id = u''
         for fname in fnames:
             try:
                 revno = int(fname)
-                yield PageRevision(self, revno, os.path.join(revisionspath, fname))
+                page_rev = PageRevision(self, revno, os.path.join(revisionspath, fname))
+                page_rev.meta[PARENTID] = parent_id
```

Comment:
for the first revision, this will have parent_id == u'' - is this how it is handled at other places also or do we have a None value or no metadata entry at all?